### ISSUE TITLE
Autosaves / saves are extremely slow

### ISSUE THREAD

**Issue by Datamats** [switchports/xash3d-switch#1]
Maybe putting the file operation on another thread would solve things?

**Comment by danfozzy** [comment]
Will overclocking help speed this up until another method is found?

**Comment by Datamats** [comment]
> Will overclocking help speed this up until another method is found?

no, overclocking will not speed up sd file system operations (at least not in any noticeable way)
overclocking not solve else anything here either
